_dashboards/geojson-regionmaps.md

```diff
@@ -15,7 +15,7 @@ OpenSearch also has a standard set of GeoJSON files to provide a vector map with
 
 ## Custom vector maps with GeoJSON
 
-If you have a specific locale that is not provided by OpenSearch Dashboards vector maps, such as a US county or US ZIP Code, you can create your own custom vector map with a GeoJSON file. To use this feature, you have to install both the [Dashboards-Maps frontend plugin](https://github.com/opensearch-project/dashboards-maps) and the [OpenSearch Geospatial backend plugin](https://github.com/opensearch-project/geospatial).
+If you have a specific locale that is not provided by OpenSearch Dashboards vector maps, such as a US county or US ZIP Code, you can create your own custom vector map with a GeoJSON file. To use this feature, you have to install both the OpenSearch Dashboards Maps front-end plugin [`dashboards-maps`](https://github.com/opensearch-project/dashboards-maps) and the OpenSearch backend plugin [`geospacial`](https://github.com/opensearch-project/geospatial).
```

Comment:
geospatial with a 't'. Maybe "geospatial backend plugin" instead of "backend plugin geospatial"?